### DISCUSSION TITLE
Is there a way to use multimedia keyboard buttons to control ncspot?

### DISCUSSION THREAD

**Discussion by alextwomey** [hrkfdn/ncspot#1539] [Q&A]
With a program like firefox, pressing the play/pause multimedia button on your keyboard will play and pause the media even if the program is not currently selected. Is there a way to do this or should I make an issue for it?

**Comment by Bettehem** [comment]
> With a program like firefox, pressing the play/pause multimedia button on your keyboard will play and pause the media even if the program is not currently selected.

This should work with ncspot too, assuming you don't have other MPRIS players running at the same time.
It is likely that pressing a media key will prioritize Firefox over ncspot since f comes before n in the alphabet.

If you want your media keys to specifically control ncspot, you can bind your media keys to run e.g. `playerctl` commands with the `-p ncspot` flag.
For example:
`playerctl -p ncspot play-pause`
